src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -806,12 +806,6 @@ namespace winrt::TerminalApp::implementation
     TerminalConnection::ITerminalConnection TerminalPage::_CreateConnectionFromSettings(Profile profile,
                                                                                         TerminalSettings settings)
     {
-        if (!profile)
```

Comment:
If we get here with a null profile, a crash report would be nice.

Reply:
If you want, you could add an `assert`, I guess